.github/workflows/ci-build.yaml

```
@@ -240,6 +240,13 @@ jobs:
             profile: minimal
             use-api: false
     steps:
+      - name: Free up disk space
+        run: |
+          printf "==> Available space before cleanup\n"
+          df -h
+          sudo rm -rf /usr/share/dotnet /usr/local/lib/android /opt/ghc /usr/local/.ghcup /opt/hostedtoolcache/CodeQL
```

```suggestion
          # these directories are not used by E2Es
          sudo rm -rf /usr/share/dotnet /usr/local/lib/android /opt/ghc /usr/local/.ghcup /opt/hostedtoolcache/CodeQL
```

add a clarifying comment

---

Added in #13670